express-api/src/services/ltsa/interfaces/ILtsaTokens.ts

```diff
@@ -0,0 +1,4 @@
+export interface ILtsaTokens {
```

Comment:
I feel like an interface this small doesn't really necessitate its own file. You could probably consolidate all these interfaces to one place.

Reply:
> I feel like an interface this small doesn't really necessitate its own file. You could probably consolidate all these interfaces to one place.

LTSA process is broken into 3 bits - get Tokens, get Title summary, get order Info. I created one interface per bit for readability purposes. If the standard is to consolidate all into one, I'm happy to do that :)